felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -664,16 +669,28 @@ func (m *bpfEndpointManager) markExistingWEPDirty(wlID proto.WorkloadEndpointID,
 }
 
 func (m *bpfEndpointManager) CompleteDeferredWork() error {
+	start := time.Now()
+	defer func() {
+		log.Infof("CompleteDeferredWork took %v", time.Since(start))
```

Comment:
See previous review, this will log every time through the main int dataplane loop whether we do any work here or not.  I don't think we can do that.  It's OK to log when the log is triggered by a change of a workload or a policy but logging more often than that is problematic